src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-content: center;
+  align-items: center;
+
+  &__star {
+    display: flex;
```

Comment:
Since '.stars__star' is a child element, there's no need to set 'display: flex' again unless you're planning to have flex items within '.stars__star'.